sunpy/data/_sample.py

```diff
@@ -166,3 +173,31 @@ def download_sample_data(overwrite=False):
             _handle_final_errors(results)
 
     return results + already_downloaded
+
+
+def _get_sample_files(filename_list, no_download=False, force_download=False):
+    sampledata_dir = _get_sampledata_dir()
+
+    fullpath = [sampledata_dir/fn for fn in filename_list]
+
+    if no_download:
+        fullpath = [fp if fp.exists() else None for fp in fullpath]
+    else:
+        to_download = zip(filename_list, fullpath)
+        if not force_download:
+            to_download = [(fn, fp) for fn, fp in to_download if not fp.exists()]
+
+        if to_download:
+            results = _download_sample_data(_BASE_URLS[0], to_download, overwrite=force_download)
+
+            # Something went wrong
```

Comment:
Could you make this comment a bit more descriptive, e.g. say at a high level what happens when something goes wrong

Reply:
This comment was a hold over from the old code, I think it should go.

Reply:
Comment updated for clarity